### ISSUE TITLE
n.lastIndexOf is not a function

### ISSUE THREAD

**Issue by RemiKalbe** [kaisermann/svelte-i18n#49]
When I init with 
```js
initialLocale: {
    navigator: true
  }
```
I have the following error:
```console
TypeError: n.lastIndexOf is not a function. (In 'n.lastIndexOf("-")', 'n.lastIndexOf' is undefined)
```
but if I do it like so:
```js
initialLocale: "en"
```
it works fine.

Here is a repro
https://codesandbox.io/s/focused-cohen-2zzp2?fontsize=14&hidenavigation=1&theme=dark

**Comment by kaisermann** [comment]
Hey @RemiKalbe 👋  You're using the `3.0-alpha` version, install the latest `v2` and you should be good to go!

I accidentally published it as the latest version and even after deprecating it people can still install it.

**Comment by RemiKalbe** [comment]
Thanks! It works 🥳

**Comment by kaisermann** [comment]
@RemiKalbe Great to know! Just noticed that I released a non up-to-date version of the `v3` branch (my bad). If you're using the `v3` released earlier today, please update to the `v3.0.1` and use `getLocaleFromNavigator()` method instead of `getClientLocale()`. Again, sorry for the confusion! 😁

**Comment by RemiKalbe** [comment]
No problem, I'll do 😉.
